src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2372,6 +2372,16 @@ bool AdaptDispatch::EndHyperlink()
     return _pConApi->PrivateEndHyperlink();
 }
 
+bool AdaptDispatch::SetWorkingDirectory(const std::wstring_view /*uri*/)
+{
+    if (_pConApi->IsConsolePty())
+    {
+        return false;
```

Comment:
Well conhost does not have tabs. So this should be OK.

Reply:
If this isn't going to be implemented in conhost, then the method shouldn't be added to `AdaptDispatch` at all. Just let it fall back to the base implementation that returns false.

Reply:
`ITermDispatch` is a virtual class. I can not fall back to the base implementation for this one.

Reply:
Sorry. My mistake. For some reason I assumed it derived from `TermDispatch` the same way `TerminalDispatch` does. It probably should, but that's not your problem.